### PR TITLE
Feat/create sessions

### PR DESCRIPTION
**Pull request by tnpb-encora**
Implemented and tested previous Caio's changes and also added an error treatment.

### CODE REVIEW

File: src/app.py [hunk 1]
```diff
@@ -1,13 +1,13 @@
 import os
 import uuid
 from langchain.text_splitter import CharacterTextSplitter
+from langchain.schema.document import Document
```
Comment (Danmcaires) [inline src/app.py:4]: There is no need, the reason Caio removed is because I imported twice

File: src/api_request.py [hunk 2]
```diff
@@ -53,7 +55,9 @@ def get_api_completion(self):
 
         # Get completion
         completion = chain.invoke({"input": self.query})
-        clean_completion = completion.split(":")[1].strip()
+        clean_completion = "-1"
```
Comment (Danmcaires) [inline src/api_request.py:58]: could we put this in an else? I think it would look better

File: src/app.py (hunk 1 above)
Comment (tnpb-encora) [reply src/app.py:4]: fixed!

File: src/api_request.py (hunk 2 above)
Comment (tnpb-encora) [reply src/api_request.py:58]: assented

File: src/app.py [hunk 3]
```diff
@@ -51,6 +51,8 @@ def create_vectorstore(llm, session_id):
     docs = [Document(page_content=x) for x in data_start]
     vectorstore = Chroma.from_documents(documents=docs, embedding=OpenAIEmbeddings(openai_api_key = OPENAI_API_KEY))
 
+    memory = ConversationBufferMemory(
+    llm=llm, memory_key="chat_history", return_messages=True)
```
[the remark this answers is not in the document]
Comment (tnpb-encora) [reply src/app.py:47]: assented

**Review by Danmcaires — APPROVED** [review]

**Pull request merged**